assembl/static2/js/app/pages/question.jsx

```diff
@@ -81,7 +81,7 @@ export function DumbQuestion(props: Props) {
               </div>
               <div className="center">
                 <h3 className="collapsed-title">
-                  <span>{`${params.questionIndex}/ ${title}`}</span>
+                  {numPosts > 1 ? <span>{`${params.questionIndex}/ ${title}`}</span> : <span>{`${title}`}</span>}
```

Comment:
No, revert this. The issue you need to fix here it to remove the question index if there is only one question (not numPosts), but we will need to a do a query of the thematic to know how many questions we have. We avoid it until now, we added the questionIndex in the url to avoid the query, but here we will need to do it.